platforms/mixin-lifecycle/src/main/java/com/dfsek/terra/lifecycle/LifecycleEntryPoint.java

```diff
@@ -0,0 +1,30 @@
+package com.dfsek.terra.lifecycle;
+
+import cloud.commandframework.execution.CommandExecutionCoordinator;
+import cloud.commandframework.fabric.FabricServerCommandManager;
+import net.minecraft.server.command.ServerCommandSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.dfsek.terra.api.command.CommandSender;
+import com.dfsek.terra.api.event.events.platform.CommandRegistrationEvent;
+
+
+public class LifecycleEntryPoint {
```

Comment:
Why is this class not final? What benefit do we get here from inheritance?

Reply:
shared code between fabric and formerly quilt before it was removed. why does it even mater if it's final. it's just an entry point, it;s only ever going to be run once

Reply:
imo it should be final and the method should just be `public static`. this class gains nothing from inheritance other than useless coupling.

Reply:
I disagree but I'll change it for the review